src/tseda/vpages/stats.py

```diff
@@ -213,12 +213,11 @@ def __panel__(self):
         windows = make_windows(self.window_size, tsm.ts.sequence_length)
         comparisons = eval_comparisons(self.comparisons.value)
 
-        sample_sets_list = (
-            self.datastore.individuals_table.selected_sample_set_indices()
-        )
+        sample_sets_dictionary = self.datastore.individuals_table.sample_sets()
+        sample_sets_list = list(sample_sets_dictionary.keys())
```

Comment:
Ok for now but I wonder if we should think about variable names here; sample_sets_list is a list of the sample set names, and sample_sets a list of lists, where each list corresponds to a sample set consisting of sample ids.

Reply:
Renamed sample_sets_list to sample_sets_ids and sample_sets to sample_sets_individuals (as it contains a list of sublists where each sublist contains the individuals of a sample set)